### PR TITLE
RRKS Basalt to TitanSMA SM recorder swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Removing BASALT (2025) from RRKS LC 20
 - Replacing with TitanSMA ~(001719)~ (002343) at same location
 - Using TitanSMA firmware version 4.5.22
 - Updating streams to continuous from `2020-02-09`

### CODE REVIEW

File: install/recorders.csv
Comment (CBurton90) [inline install/recorders.csv:1014]: Seems like the azimuth needs an update to `274`.

https://github.com/GeoNet/tickets/issues/16492#issuecomment-2381634041

Comment (Thomas-Benson) [reply install/recorders.csv:1014]: updated

**Comment by Thomas-Benson** [comment]
TitanSMA installed at a new orientation of `274` degrees

**Review by CBurton90 — APPROVED** [review]

**Review by mnaguit — COMMENTED** [review]
From the ticket, the serial number of the newly-installed TitanSMA is `002343`?
Thanks @Thomas-Benson @CBurton90

**Comment by Thomas-Benson** [comment]
@mnaguit you're correct, it should be `002343`. the SN was wrong in the ticket when I started the PR 🫠

**Review by mnaguit — APPROVED** [review]
Thanks @Thomas-Benson 👍

**Review by CBurton90 — APPROVED** [review]

**Comment by Thomas-Benson** [comment]
Sorry folks, need one last approval after resolving merge conflicts

**Review by mnaguit — APPROVED** [review]

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**
